src/vstest.console/CommandLine/Executor.cs

```diff
@@ -93,7 +93,16 @@ internal int Execute(params string[] args)
         {
             this.testPlatformEventSource.VsTestConsoleStart();
 
-            this.PrintSplashScreen();
+            // If User specifies --nologo via dotnet, donot print splat screen
+            if (args != null && args.Length !=0 && args.Contains("--nologo"))
```

Comment:
Move this logic to function.

Reply:
Case insensitive?

Reply:
we are controlling this argument, can make case insensitive, but didn't need it now